AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

```diff
@@ -10,7 +10,7 @@ import AWSClientRuntime
 
 public class AmplifyAWSServiceConfiguration {
     public static let version = "2.0.0"
-    public static let platformName = "amplify-ios"
```

Comment:
Does anything reporting/metrics-related rely on `amplify-ios`?

Reply:
Verified with the team that this does not break anything.

Reply:
Received go ahead from reporting to update this value.